### ISSUE TITLE
Facepile A11y Pass

### ISSUE THREAD

**Issue by TestAccessibilityGIT** [OfficeDev/office-ui-fabric-react#7534]
Tracking all of the issues brought up with the 1/4 accessibility pass on Facepile control 
Details on individual bugs can be found on [SharePoint](https://microsoft-my.sharepoint.com/:x:/r/personal/v-amaver_microsoft_com/_layouts/15/Doc.aspx?sourcedoc=%7Bfd43fcc5-515d-47d8-946d-0d4862a56278%7D&action=default&uid=%7BFD43FCC5-515D-47D8-946D-0D4862A56278%7D&ListItemId=1647&ListId=%7B395E1ECB-5002-440E-81C5-6A090E98077C%7D&odsp=1&env=prodbubble) 
  
 - [ ] Bug1: [Accessibility][Programmatic access]: Certain ARIA roles must be contained by particular parents. 
 - [ ] Bug2: [Accessibility][Keyboard navigation]: User is not able to access all facepile controls using keyboard.  
 - [ ] Bug3: [Accessibility][Usable]: In scan mode narrator doesn't announce role and state of the 'Overflow Button Type',  'Persona Size' list box.  
 - [ ] Bug4: [Accessibility][Usable]: Narrator doesn't announce the slider value along with the control name while changing the 'Number of persona' slider using arrow keys. 
 - [ ] Bug5: [Accessibility][Functional]: No option is given to remove the facepile control present under 'Facepile with face adding functionality' section. 
 - [ ] Bug6: [Accessibility][Screen reader]: Narrator announces same position number for all facepile controls.  
 - [ ] Bug7: [Accessibility][Usable]: Narrator scan mode focus is not visible on facepile controls as screen gets scroll down once user navigates on them.   
 - [ ] Bug8: [Accessibility][Programmatic access]: Screen reader doesn't give any feedback while adding facepiles using 'Add friend' button. 
 
 
 Component: Facepile 
@micahgodbolt

**Comment by tabrumle** [comment]
Is there an estimate on when this work, specifically bug 1 and 6, would be funded? There is an A11yMAS bug in office that we need fixed by the end of the month. @jspurlin for visibility.

**Comment by micahgodbolt** [comment]
@markionium @mtennoe is this something you can take a look at?

**Comment by tabrumle** [comment]
The fix seems to be smaller than I had thought. I've sent out a PR with the possible change.

**Comment by Markionium** [comment]
Thanks for taking a look @tabrumle. @micahgodbolt I'll have a look this week.

**Comment by tabrumle** [comment]
I agree that Bug 1 is no longer applicable. The personas should now all housed under the right li elements now. I haven't seen Bug 2 in the office use case since we always have an onClick action associated with the FacepileButtons and are able to access them. I do agree though that it would seem strange to navigate to uninteractive controls. Narrator users can also just use CAPS+arrows to read the content so I don't think there is no need to add further keyboard navigation there.

**Comment by Markionium** [comment]
@tabrumle is your immediate concern fixed? It'd be nice to know with regard to how we should prioritise this in our team.

**Comment by tabrumle** [comment]
The concern was fixed with PR [9823](https://github.com/OfficeDev/office-ui-fabric-react/pull/9823). I am not seeing this change on the Fabric website though.

**Comment by jdhuntington** [comment]
resolving as dupe, opened separate issues for each bug.
